README.md

```diff
@@ -1 +1,207 @@
+
+#### Branches
+
+Once you have the initial tasks distributed as issues in your repo, your development should involve **separate branches** **based** on the **tasks** you do. Making a **branch** means that you **have a safe copy** of the code to work on in a **sandbox** style environment. This ensures that **if you** do something on your branch that **messes** up the **already working code** in the main branch, your **mistake** would **not** **affect** the **main code** and you can start over to ensure you don't encounter the same problem.
+
+To **make** a branch, you open the main page of the repository (Code).
+1. Click on the branch (<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path d="M9.5 3.25a2.25 2.25 0 1 1 3 2.122V6A2.5 2.5 0 0 1 10 8.5H6a1 1 0 0 0-1 1v1.128a2.251 2.251 0 1 1-1.5 0V5.372a2.25 2.25 0 1 1 1.5 0v1.836A2.493 2.493 0 0 1 6 7h4a1 1 0 0 0 1-1v-.628A2.25 2.25 0 0 1 9.5 3.25Zm-6 0a.75.75 0 1 0 1.5 0 .75.75 0 0 0-1.5 0Zm8.25-.75a.75.75 0 1 0 0 1.5.75.75 0 0 0 0-1.5ZM4.25 12a.75.75 0 1 0 0 1.5.75.75 0 0 0 0-1.5Z"></path></svg>) dropdown menu and click **"View all branches"**
+2. Create a big green button that says **"New Branch"**
+3. Give the branch a descriptive name
+
+To **delete** a branch, open the tab with all branches and click on the bin (<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16"><path d="M16 1.75V3h5.25a.75.75 0 0 1 0 1.5H2.75a.75.75 0 0 1 0-1.5H8V1.75C8 .784 8.784 0 9.75 0h4.5C15.216 0 16 .784 16 1.75Zm-6.5 0V3h5V1.75a.25.25 0 0 0-.25-.25h-4.5a.25.25 0 0 0-.25.25ZM4.997 6.178a.75.75 0 1 0-1.493.144L4.916 20.92a1.75 1.75 0 0 0 1.742 1.58h10.684a1.75 1.75 0 0 0 1.742-1.581l1.413-14.597a.75.75 0 0 0-1.494-.144l-1.412 14.596a.25.25 0 0 1-.249.226H6.658a.25.25 0 0 1-.249-.226L4.997 6.178Z"></path><path d="M9.206 7.501a.75.75 0 0 1 .793.705l.5 8.5A.75.75 0 1 1 9 16.794l-.5-8.5a.75.75 0 0 1 .705-.793Zm6.293.793A.75.75 0 1 0 14 8.206l-.5 8.5a.75.75 0 0 0 1.498.088l.5-8.5Z"></path></svg>) symbol at the end of the line with the branch you want to delete.
+
+#### Commits
+
+Commits are a way to update the repository with the changes you made. Each commit has information about who made the change, what changes were made, when the change was made, and a message that describes the changes (and/or if it refers to an [Issue](#issues)).
+
+Commits are very important part of collaboration and should follow an agreed on convention for the team. Each commit also offers a way to organize the changes because it shows a very short concise description of the change made. As such, make sure to commit frequently and that each commit solves only one issue. As with Issues, your commits should be atomic (contains only one change).
+
+##### Guidelines for good commit messages
```

Comment:
give an example in each category if possible.

Reply:
I added an example of commit `Feat: implement user input parsing for calculator to split string into tokens (Issue #1)`. Would this be enough or did you mean for me to add examples for other parts of the README, too?